src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/PdfDocumentGenerationServiceGotenbergIntegrationTest.kt

```diff
@@ -25,15 +26,15 @@ class PdfDocumentGenerationServiceGotenbergIntegrationTest {
           <body>
               <center>
                 <h1>Test landscape pdf</h1>
```

Comment:
This h1 is misleading, isn't it?  More "Test with logo" than "Test landscape pdf"?